testing_and_setup/compass/manage_regression_suite.py

```diff
@@ -495,14 +534,15 @@ def summarize_suite(suite_tag):  # {{{
         if args.setup:
             print("\n")
             print("Setting Up Test Cases:")
+            testcases = get_test_case_procs(suite_root)
             setup_suite(suite_root, args.work_dir, args.model_runtime,
                         args.config_file, args.baseline_dir, args.verbose)
-            summarize_suite(suite_root)
+            summarize_suite(testcases)
```

Comment:
The changes require doing the parsing that was previously in `summarize_suite()` before doing `setup_suite()` so we can error out if the prerequisites are not present.  The info from this parsing is stored in nested `dict`s that will likely be useful for future efforts for parallelization.